atomate/vasp/workflows/presets/core.py

```diff
@@ -53,6 +53,34 @@ def wf_bandstructure(structure, c=None):
 
     return wf
 
+def wf_scan_opt(structure,c={}):
```

Comment:
does this need to be its own workflow? Can it be a powerup to other workflows?

I don't want to end up with wf_scan_static, wf_scan_elastic_tensor, wf_scan_dielectric, etc. - and then repeat that with every other functional.

Reply:
Right now, only the structure optimization step benefits from the extra logic embedded in custodian. So far, I haven't needed the electronic preconditioning for static calculations. For every other type of calculation, I don't know if it will work or is needed. Do we want to examine the custodian logic? Should there be a default GGA precondition flag or something like that in RunVaspCustodian rather than the current method of having a special custodian job? I'm going off my current experience with SCAN and don't see a point in putting in something into atomate that someone has actually tried and tested.